src/common/api/order-builder.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+
+export default class OrderBuilder {
+    private order: string[] = [];
```

Comment:
`private _order` ?

Reply:
Current tslint config does not allow that.